client/src/main/java/com/aws/greengrass/cli/commands/DeploymentCommand.java

```diff
@@ -55,7 +56,8 @@ public DeploymentCommand(NucleusAdapterIpc nucleusAdapterIpc) {
      @CommandLine.Option(names = {"-a", "--artifactDir"}, paramLabel = "Artifacts directory") String artifactDir,
      @CommandLine.Option(names = {"--runWith"}, paramLabel = "Component user and/or group") Map<String, String> runWithOptions,
      @CommandLine.Option(names = {"--systemLimits"}, paramLabel = "Component system resource limits") String systemLimits,
-     @CommandLine.Option(names = {"-c", "--update-config"}, paramLabel = "Component configuration") String configUpdate)
+     @CommandLine.Option(names = {"-c", "--update-config"}, paramLabel = "Component configuration") String configUpdate,
+     @CommandLine.Option(names = {"-p", "--failure-policy"}, paramLabel = "Component configuration") FailureHandlingPolicy failureHandlingPolicy)
```

Comment:
need to fix `paramLabel`

Reply:
is this input of FailureHandlingPolicy case sensitive? Should we consider taking it in as a String as handle it ourselves?

Reply:
--failure-handling-policy to be consistent with cloud?